lobsterpy/plotting/__init__.py

```diff
@@ -226,3 +236,427 @@ def _broaden(energies: np.ndarray, population: np.ndarray, sigma=None, cutoff=4.
         kernel = norm.pdf(kernel_x, scale=sigma)
 
         return convolve(population, kernel, mode="same") / kernel.sum()
+
+
+class InteractiveCohpPlotter(CohpPlotter):
+    """
+    Interactive COHP plotter to view all relevant / multiple COHPs in one figure.
+    """
+
+    def add_all_relevant_cohps(
+        self, analyse: Analysis, suffix: str = "", label_resolved: bool = True
+    ):
```

Comment:
`) -> None:` is helpful if there is no return value

Reply:
Added the return as None 😄